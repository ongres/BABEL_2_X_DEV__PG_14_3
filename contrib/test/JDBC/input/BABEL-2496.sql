use master;
go

drop schema if exists non_exist_schema;
go

drop schema non_exist_schema;
go

create schema s2496;

drop schema if exists s2496;
go

drop schema s2496;
go

drop schema if exists s2496;
go

drop schema s2496;
go
