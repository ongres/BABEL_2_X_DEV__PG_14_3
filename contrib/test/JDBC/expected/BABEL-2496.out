use master;
go

drop schema if exists non_exist_schema;
go

drop schema non_exist_schema;
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: schema "master_non_exist_schema" does not exist)~~



create schema s2496;
drop schema if exists s2496;
go

drop schema s2496;
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: schema "master_s2496" does not exist)~~


drop schema if exists s2496;
go

drop schema s2496;
go
~~ERROR (Code: 33557097)~~

~~ERROR (Message: schema "master_s2496" does not exist)~~

